.editorconfig

```diff
@@ -0,0 +1,14 @@
+[*]
+indent_style = space
+indent_size = 2
+end_of_line = lf
+charset = utf-8
+trim_trailing_whitespace = true
+insert_final_newline = true
+
+[*.java]
+indent_size = 4
```

Comment:
so far I think we've been using 4 spaces mainly in Java even if currently we have a mix. I would prefer spaces for all.